identity/app/model/EmailSubscriptions.scala

```diff
@@ -97,6 +97,7 @@ object EmailSubscriptions {
       frequency = "Tuesday mornings",
       listId = "3698",
       subscribedTo = subscribedListIds.exists{ x => x == "3698" },
+      signupPage = Some("https://www.theguardian.com/politics/2016/may/31/eu-referendum-morning-briefing-sign-up"),
```

Comment:
This means dev and code environments will link to live. It would be better to have a relative path?